src/components/UI/Text/TimeInput.vue

```diff
@@ -15,6 +15,8 @@
         class="w-12"
         :maxLength="2"
         :boxStyling="[ defaultBoxClass, hourInputInvalidClass ]"
+        :isDisabled="isDisabled"
+        v-tooltip.bottom="disabledInputTooltip"
```

Comment:
time boxes will be disabled when plio is published and a tooltip will be displayed